lib/galaxy/datatypes/converters/bcf_to_bcf_bgzip_converter.xml

```diff
@@ -0,0 +1,14 @@
+<tool id="CONVERTER_bcf_to_bcf_bgzip" name="Convert BCF to BCF_BGZIP" version="0.0.1" hidden="false">
+<!--  <description>__NOT_USED_CURRENTLY_FOR_CONVERTERS__</description> -->
+  <command interpreter="python">bcf_to_bcf_bgzip_converter.py '$input1' '$output1'</command>
```

Comment:
same here: you could write something like `bcftools view "$input1" -o "$output1" -O b`